types.go

```diff
@@ -241,3 +243,13 @@ type ThreeDS struct {
 	Version          string // 3DS Version
 	XID              string // Transaction ID from authentication processing (for 3DS1)
 }
+
+type ResultType string
+
+const (
+	ResultTypeSuccess      ResultType = "Approved"
```

Comment:
curious: so we are not going to use it for now?

Reply:
Yes its a bit redundant with the `Success` field but would be much bigger effort to completely remodel the auth response and make consistent across the various PSPs.